ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,72 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
+- fire_alarm_status
+- fire_fault_status
+- makeup_water_high_tank_percentage_status
+- makeup_water_low_tank_percentage_status
+- max_cooling_air_flowrate_setpoint
+- min_cooling_air_flowrate_setpoint
+- pump_fault_status
+- pump_run_status
```

Comment:
is this an integral pump or just one for a system? we model pumps individually unless they are integral to the device (such as a circulation pump on a boiler).

Reply:
it is a Grease Waste Interceptor pump

Reply:
so its just a pump, and you're modeling it as a pump? if so, `run_status ` and `run_command` are sufficient.